samples/trace/index.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// [START opentelemetry_trace_samples]
+'use strict';
+
+// [START opentelemetry_trace_import]
+const opentelemetry = require('@opentelemetry/api');
+const {NodeTracerProvider} = require('@opentelemetry/node');
+const {SimpleSpanProcessor} = require('@opentelemetry/tracing');
+const {
+  StackdriverTraceExporter,
```

Comment:
Since this is a new library, does it make sense to export an object referring to a historic product name?

Reply:
Good point, let me check what other languages are doing.

Reply:
Worst case, we could keep it for backward compatibility with the old exporter and re-export a more suitable name, like CloudTraceExporter. ..and yes, we should sync this across languages.

Reply:
Changed to `TraceExporter` in samples and actual code (please review #24).